includes/phing/props/docker.props

```diff
@@ -0,0 +1,4 @@
+db.host = mysql
```

Comment:
Needs header like other props files.

Reply:
I will fix it, but in a future release we will need to review them all. Check the todos and decide a uniform sintaxe.